tests/unittests/Test-gw.cpp

```diff
@@ -0,0 +1,770 @@
+// SPDX-FileCopyrightText: 2024 Lisa Biermann, Margarete Mühlleitner, Rui
+// Santos, João Viana
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include <catch2/catch_approx.hpp>
+#include <catch2/catch_test_macros.hpp>
+
+using Approx = Catch::Approx;
+
+#include <BSMPT/bounce_solution/action_calculation.h>
+#include <BSMPT/gravitational_waves/gw.h>
+#include <BSMPT/minimizer/Minimizer.h>
+#include <BSMPT/minimum_tracer/minimum_tracer.h>
+#include <BSMPT/models/ClassPotentialOrigin.h> // for Class_Potential_Origin
+#include <BSMPT/models/IncludeAllModels.h>
+#include <BSMPT/models/ModelTestfunctions.h>
+#include <BSMPT/transition_tracer/transition_tracer.h>
+#include <BSMPT/utility/Logger.h> // for Logger Class
+#include <fstream>
+
+TEST_CASE("Test I_\alpha", "[gw]")
```

Comment:
```suggestion
TEST_CASE("Test I_alpha", "[gw]")
```

This caused my IDE to render the `\a` as a small telephon icon and the test was shown failing